src/prefect/deployments/recipes/docker-git/prefect.yaml

```diff
@@ -36,4 +36,4 @@ deployments:
       name: null
       work_queue_name: null
       job_variables: 
-        image: "{{ build_image.image_name }}"
\ No newline at end of file
+        image: "{{ build_image.image }}"
```

Comment:
sorry, why is it changed to `image`? Isn't it coming from:

```yaml
build:
  - prefect_docker.deployments.steps.build_docker_image:
      id: "build_image"
      requires: "prefect-docker>=0.3.0"
      image_name: "{{ image_name }}"
      tag: "{{ tag }}"
      dockerfile: "{{ dockerfile }}"
      push: true
```

Reply:
`image` is output from `build_docker_image` which is the `image_name` and `tag` concatenated with a `:`. We made that change to make it more natural when passing `image` into work pool job variables.

Reply:
just a thought, do you think it would be worth at some point adding a comment under the `build` section that mentions that is an output?

Reply:
Makes sense! I added a comment for clarity.